go.mod

```diff
@@ -24,12 +25,14 @@ require (
 	github.com/tidwall/gjson v1.14.1
 	golang.org/x/net v0.5.0
 	google.golang.org/grpc v1.53.0
+	gopkg.in/yaml.v3 v3.0.1
```

Comment:
I'd like to use `gopkg.in/yaml.v3` since the ` github.com/ghodss/yaml` seems no longer maintained.

https://github.com/ghodss/yaml/issues/80

@andreyvelich  Do you have any reason to avoid using `gopkg.in/yaml.v3`?

Reply:
I see, makes sense, should we use `sigs.k8s.io/yaml` library as other projects to replace `github.com/ghodss/yaml` ?
I remember we had problems with YAML parsing using `gopkg` lib.

I think, we also should remove this dependency from old Katib UI backend to remove this legacy library: https://github.com/kubeflow/katib/blob/master/pkg/ui/v1beta1/backend.go#L24C9-L24C25.
We might need to work to remove old UI from our Katib repository completely .
cc @johnugeorge

Reply:
> I see, makes sense, should we use sigs.k8s.io/yaml library as other projects to replace github.com/ghodss/yaml ?
I remember we had problems with YAML parsing using gopkg lib.

I see. I will try to use `sigs.k8s.io/yaml`.

Reply:
Done.